automatic/qbittorrent/update.ps1

```diff
@@ -32,7 +32,7 @@ function global:au_GetLatest {
   $version64 = $url64 -split '[_]' | select -Last 1 -Skip 2
 
   if ($version -ne $version64) {
-    throw "32-bit and 64-bit version do not match. Please investigate."
+    throw "32-bit and 64-bit version do not match. Please contact the package manager."
```

Comment:
You know this is a message to the package maintainers, right?
Basically now you're telling package maintainers to contact the package managers.
Please revert this line, as it doesn't really make sense.